code-for-canada/project-thundercat
# [Inbox - Starting Test] Switch to background tab when the test starts

# Description

When starting the test, there is now an automatic switch from _Instruction_ to _Background_ tab.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Starting the test:**

![image](https://user-images.githubusercontent.com/23021242/56960214-11014e80-6b1e-11e9-8be0-5c7a95bf0fd0.png)

![image](https://user-images.githubusercontent.com/23021242/56960239-224a5b00-6b1e-11e9-9c9e-7a3450e841aa.png)

**_Background_ tab is now selected automatically:**

![image](https://user-images.githubusercontent.com/23021242/56960266-3a21df00-6b1e-11e9-9e39-9e6ebec20191.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Enter the eMIB.
3.  Start the test and make sure that the tab is switched to _Background_ automatically.

# Checklist
